### ISSUE TITLE
[Components and pattern standards] Design components and patterns in use are outdated.  (04.08.1)

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#59258]
#### VFS team name
Benefits Disability Experience Team

#### VFS product name
Disability Comp Application (21-526 EZ), Add Self Assessment Part A form information to BDD workflow

#### VFS feature name
Add Self Assessment Part A form information to BDD workflow

#### Point of Contact/Reviewers
Allison Christman - @allison0034 - Design
Allison Christman - @allison0034 - Design
Brian DeConinck - @briandeconinck - Accessibility

*For more information on how to interpret this ticket, please refer to the [Anatomy of a Staging Review issue ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/Anatomy-of-a-Staging-Review-Issue-ticket.2060320997.html) guidance on Platform Website.

---

### Platform Issue
Design components and patterns in use are outdated. 

### Issue Details
The "Is this the form I need?" page is using conditional logic to display a wizard. The [wizard](https://design.va.gov/patterns/wizards) is deprecated.

The "Supporting evidence" page is using conditional logic to display a wizard. The wizard is deprecated.

The wizard pattern is particularly challenging for assistive technology users, especially screen reader users. As currently coded, the start page wizard doesn't announce to screen reader users its determination of what form to use. Right now screen reader users would have to hunt around to find that text.

### Link, screenshot or steps to recreate
https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/working-with-vsp/vsp-collaboration-cycle/staging-review-images/56612-Allison-040801.png
https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/working-with-vsp/vsp-collaboration-cycle/staging-review-images/56612-Allison0949891-2.png

### VA.gov Experience Standard
[Category Number 04, Issue Number 08](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/VA.gov-experience-standards.1683980311.html)

### Other References
WCAG SC 3.2.4 AA

### Platform Recommendation
Go to 1 question per page instead of displaying additional questions on a single page based on the answer.

Go to 1 question per page instead of displaying additional questions on a single page based on the answer.

---

### VFS Guidance

- Close the ticket when the issue has been resolved or validated by your Product Owner
- If your team has additional questions or needs Platform help validating the issue, please comment on the ticket
- Some feedback provided may be out of scope for your iteration of the product, however, Platform's OCTO leadership has stated that all identified issues need to be documented and it is still your responsibility to resolve the issue.
- If you do not believe that this Staging Review issue ticket is the responsibility of your team, comment below providing an explanation and who you believe is responsible. Please tag the Point of Contact/Reviewers. Governance team will research and will follow up.

**Comment by lydiahooper** [comment]
We will eliminate these wizards as part of the intro pages redesign and 4142 redesign. I'll close this ticket when those changes are made.
